package.json

```diff
@@ -1,112 +1,98 @@
-    "coverage": "gulp coveralls",
-    "generate-docs": "node ./build/commands-yml/parse.js",
-    "zip": "zip -qr appium.zip .",
-    "upload": "gulp github-upload",
-    "zip-and-upload": "npm run zip && npm run upload"
+    "reinstall": "npm run clean && npm install",
+    "start": "node ./packages/appium",
+    "test": "lerna exec gulp once",
+    "test:appium": "lerna exec --scope=appium gulp once",
+    "test:base-driver": "lerna exec --scope=@appium/base-driver gulp once",
+    "test:doctor": "lerna exec --scope=@appium/doctor gulp once",
+    "test:e2e:appium": "lerna exec --scope=appium gulp e2e-test",
+    "test:e2e:base-driver": "lerna exec --scope=@appium/base-driver gulp e2e-test",
+    "test:e2e:fake-driver": "lerna exec --scope=@appium/fake-driver gulp e2e-test",
+    "test:e2e:gulp-plugins": "lerna exec --scope=@appium/gulp-plugins gulp e2e-test",
+    "test:e2e:support": "lerna exec --scope=@appium/support gulp e2e-test",
+    "test:fake-driver": "lerna exec --scope=@appium/fake-driver gulp once",
+    "test:gulp-plugins": "lerna exec --scope=@appium/gulp-plugins gulp once",
+    "test:support": "lerna exec --scope=@appium/support gulp once",
+    "test:test-support": "lerna exec --scope=@appium/test-support gulp once",
+    "watch": "lerna exec watch"
   },
   "pre-commit": [
     "precommit-msg",
-    "precommit-test"
+    "precommit-lint"
   ],
+  "lint-staged": {
+    "*.js": "eslint --fix"
+  },
   "devDependencies": {
+    "@appium/eslint-config-appium": "file:./packages/eslint-config-appium",
     "@appium/fake-plugin": "^1.0.0",
-    "appium-fake-driver": "^2.0.1",
-    "appium-gulp-plugins": "^5.2.1",
+    "@babel/core": "^7.0.0",
+    "@babel/eslint-parser": "^7.14.3",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-private-methods": "^7.0.0",
+    "@babel/plugin-syntax-bigint": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/register": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
+    "appium-base-driver": "^7.6.0",
+    "asyncbox": "^2.8.0",
+    "babel-eslint": "^10.0.3",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-source-map-support": "^2.0.1",
     "chai": "4.x",
     "chai-as-promised": "7.x",
-    "eslint-config-appium": "^4.0.1",
+    "colors": "^1.4.0",
+    "eslint-find-rules": "^3.6.1",
     "fancy-log": "^1.3.2",
+    "finalhandler": "^1.1.1",
+    "get-port": "^5.1.1",
     "gulp": "^4.0.0",
     "handlebars": "^4.2.0",
+    "lerna": "^4.0.0",
+    "lint-staged": "^11.0.0",
+    "mjpeg-server": "^0.3.0",
     "mocha": "^8.0.1",
     "pre-commit": "1.x",
+    "querystring": "^0.2.0",
+    "serve-static": "^1.13.1",
     "sinon": "^9.0.0",
     "validate.js": "^0.13.0",
+    "wd": "^1.14.0",
```

Comment:
note: this is something we don't want to depend anymore :)

Reply:
yeah, need to update fake-driver.  that will be a separate PR